### PR TITLE
[12.0][MIG] web_widget_timepicker

### PR DESCRIPTION
**Pull request by dalonsod**
Standard migration

### CODE REVIEW

File: web_widget_timepicker/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (NL66278) [inline web_widget_timepicker/__init__.py]: Make it a habit to turn all license lines to https://

File: web_widget_timepicker/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 Vividlab (<http://www.vividlab.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Web Timepicker Widget",
+    "version": "12.0.1.0.0",
+    "author": u"VividLab, "
```
Comment (NL66278) [inline web_widget_timepicker/__manifest__.py]: No need for any `u`'s. This is python 3.x and therefore strings are Unicode by definition.

**Review by NL66278 — APPROVED** [review]
:+1: Some nitpicks but LGTM.

**Review by lfreeke — APPROVED** [review]
functional test

File: web_widget_timepicker/__manifest__.py (hunk 2 above)
Comment (dalonsod) [reply web_widget_timepicker/__manifest__.py]: Done

File: web_widget_timepicker/__init__.py (hunk 1 above)
Comment (dalonsod) [reply web_widget_timepicker/__init__.py]: Done for every *.py

**Comment by dalonsod** [comment]
@OCA/web-maintainers is this PR ready to be merged? Thanks!

File: web_widget_timepicker/README.rst [hunk 3]
```diff
@@ -83,6 +83,7 @@ Contributors
 * Michael Fried <Michael.Fried@vividlab.de>
 * Kaushal Prajapati <kbprajapati@live.com>
 * Alexandre Díaz <dev@redneboa.es>
+* David Alonso <david.alonso@solvos.es>
```
Comment (simahawk) [inline web_widget_timepicker/README.rst:86]: FTR no-brainer migrations normally don't allow this.

Comment (dalonsod) [reply web_widget_timepicker/README.rst:86]: Thanks, I'll keep it mind in the future

**Pull request merged**